ykrt/src/compile/jitc_yk/aot_ir.rs

```diff
@@ -546,7 +655,21 @@ impl Module {
         *self.var_names_computed.borrow_mut() = true;
     }
 
-    /// Fill in the function index of local variable operands of instructions.o
+    pub(crate) fn func_index(&self, find_func: &str) -> Option<usize> {
+        // OPT: create a cache in the Module.
+        for (f_idx, f) in self.funcs.iter().enumerate() {
```

Comment:
I think this function can become just `self.funcs.iter().enumerate().find(|x| x.name == find_func).0` or similar.

Reply:
Fixed in 8284b7c11508a06eb6cd41ba90749b926c6045ca